pkgs/development/python-modules/scim2-client/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchPypi,
+  hatchling,
+  scim2-models,
+  pytestCheckHook,
+  portpicker,
+  pytest-httpserver,
+  pytest-asyncio,
+  scim2-server,
+  httpx,
+  werkzeug,
+}:
+
+buildPythonPackage rec {
+  pname = "scim2-client";
+  version = "0.5.1";
+
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "scim2_client";
+    hash = "sha256-g2RR+Ruvjw88cGHcwEPoktTmB8VcWAPnea3BErS8JyI=";
+  };
+
+  build-system = [ hatchling ];
+
+  dependencies = [ scim2-models ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    portpicker
+    pytest-httpserver
+    pytest-asyncio
+    scim2-server
+    werkzeug
+  ] ++ optional-dependencies.httpx;
+
+  # Werkzeug returns 500, didn't deem it worth it to investigate
```

Comment:
I am the maintainer of the python-scim collection too, so if you need help for this, let me know.